tutormfe/templates/mfe/build/mfe/Dockerfile

```diff
@@ -68,6 +68,13 @@ RUN {% if is_buildkit_enabled() %}--mount=type=cache,target=/root/.npm,sharing=s
 {{ patch("mfe-dockerfile-post-npm-install-{}".format(app_name)) }}
 COPY --from={{ app_name }}-src / /openedx/app
 COPY --from={{ app_name }}-i18n /openedx/app/src/i18n/messages /openedx/app/src/i18n/messages
+
+# Whenever a new MFE supports Atlas, it should be added to this list.
+# When all MFEs support Atlas, this if-statement should be removed.
+{% if app_name in ["communications"] %}
+RUN make OPENEDX_ATLAS_PULL=true pull_translations
```

Comment:
Are you familiar with the i18-merge.js script from line 53? It's a script that allows anyone to override translation strings. If I understand correctly, running `pull_translations` here would override this behaviour, thus preventing users from adding their custom translations. Can you confirm? Can we run Atlas in a way that is compatible with this existing feature?

Also, is atlas supported in Palm? If not, this PR should be opened on top of nightly.

Reply:
> Are you familiar with the i18-merge.js script from line 53? It's a script that allows anyone to override translation strings. If I understand correctly, running pull_translations here would override this behaviour, thus preventing users from adding their custom translations. Can you confirm? Can we run Atlas in a way that is compatible with this existing feature?

I'm familiar with the script and I should have been clearer in the pull request description that we're going to intentionally not support `i18n-merge.js` and `atlas pull` at the same time in this iteration. This will affect the communications MFE which has _no translations_ at the moment so there's no step backward.

Once this PR is merged and used, we'll work on supporting the `i18n-merge.js` if needed. Our understanding that Atlas provides a superior workflow which makes `i18n-merge.js` not needed -- at least as we understand it.

This has been confirmed with @arbrandes -- tagging him just in case I missed anything.

Is what I'm mentioning a blocker to merging this PR?

> Also, this change is going to break the build for all users who do not run your fork of the communications app, right? We should definitely avoid this, one way or another.

No, of course not. The fork is being used merely for reviewing this pull request, we are fixing this on upstream and it's pending merge: 

 - Palm: https://github.com/openedx/frontend-app-communications/pull/156
 - Quince: Pull request will be created.

Reply:
@regisb @arbrandes The https://github.com/openedx/frontend-app-communications/pull/156 pull request has been merged into `palm.master`. We need the following updates to make Tutor MFE support Atlas without needing a fork:

 - Tag `open-release/palm.4` on the communications MFE.
 - Update Tutor open edx version `open-release/palm.2` --> `open-release/palm.4`

This is needed to fix the i18n for the communications app on Palm.

Once this is merged, it will also cause Quince to be supported. The good news that Atlas support already exists in the quince branch: https://github.com/openedx/frontend-app-communications/blob/3e7ea6a3e5d5c35bf508fc7fab067fc5be836e75/Makefile#L52-L62

Reply:
> This will affect the communications MFE which has no translations at the moment so there's no step backward.

Well, it will be a breaking change for users who implemented their own translation strings for the communications MFE. It's a niche use case though, so maybe we can just ignore it.

I agree that Atlas should be a much better solution than that weird i18s-merge.js script. Still, we need to provide users a way to customise translation strings. How are we going to do that in the tutor-mfe plugin?

> Tag open-release/palm.4 on the communications MFE.

Publishing a new release tag is a big deal. Do you really need Atlas in Palm? If yes we can ask the BTR to publish a new tag. But maybe it would be sufficient to support Atlas in Quince? You tell me.

Reply:
> Well, it will be a breaking change for users who implemented their own translation strings for the communications MFE. It's a niche use case though, so maybe we can just ignore it.
> I agree that Atlas should be a much better solution than that weird i18s-merge.js script. Still, we need to provide users a way to customise translation strings. How are we going to do that in the tutor-mfe plugin?

For the communications MFE i18n-merge.js is mostly unused due to completely missing translations. Based on the feedback we've had, no one translated it except for Pierre Mailhot which doesn't use the i18n-merge. For other MFEs, yes it makes sense to add this support.

We've discussed this issue in call with Adolfo and we decided that this iteration won't support i18n merge.

Please let me know if that's still a blocker.

> Publishing a new release tag is a big deal. Do you really need Atlas in Palm? If yes we can ask the BTR to publish a new tag. But maybe it would be sufficient to support Atlas in Quince? You tell me.

It's a big deal indeed. Let's refocus our efforts into Quince. I'll check with @sambapete on the idea of skipping Palm for Tutor MFE support, which would leave the Palm users no way to support Atlas except by cherry-picking this pull request. Which isn't pretty bad as long as we land it into Quince.

Reply:
Thanks @sambapete, Quince it is! We're close to do so -- hopefully :)

Reply:
+1 for Quince.

We can still do Palm.4, but it should come _after_ getting Quince up and running.